tcms/testcases/migrations/0010_remove_bug.py

```diff
@@ -26,7 +26,7 @@ class Migration(migrations.Migration):
 
     operations = [
         # copy the data from the related model
-        migrations.RunPython(forward_copy_data),
+        migrations.RunPython(forward_copy_data, migrations.RunPython.noop),
```

Comment:
This one also leads to data loss:

1) In the forwards function store all Bug objects as json files under /tmp like shown above
2) Then keep track of the newly created LinkReference objects (IDs is fine), store them on disk too (json is fine)
3) When going backwards all LinkReference's with IDs in the list should be removed
4) Bug objects should be restored from disk.